docs/releases/5.1.1.txt

```diff
@@ -18,3 +18,7 @@ Bugfixes
   children). A new :class:`~django.contrib.auth.forms.AdminUserCreationForm`
   including this field was added, isolating the feature to the admin where it
   was intended (:ticket:`35678`).
+
+* Restored, following a regression in Django 5.1, the ability to override the
+  timezone and role setting behavior within
+  ``DatabaseWrapper.init_connection_state()`` (:ticket:`35688`).
```

Comment:
Shall we explicitely limit this release note to the Postgresql backend? I don't see any other backend, not even base, that define a method to define the role behavior.